.github/workflows/daily.yml

```diff
@@ -1,4 +1,4 @@
-name: SchedulerBundle - Daily
+name: Daily
```

Comment:
Requires to change badge's URL on README

Reply:
Fixed